x-pack/metricbeat/module/mssql/transaction_log/_meta/data.json

```diff
@@ -0,0 +1,54 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "event": {
+        "dataset": "mssql.transaction_log",
+        "duration": 115000,
+        "module": "mssql"
+    },
+    "metricset": {
+        "name": "transaction_log"
+    },
+    "mssql": {
+        "database": {
+            "id": 1,
+            "name": "master"
+        },
+        "transaction_log": {
+            "space_usage": {
+                "since_last_backup": {
+                    "bytes": 135168
+                },
+                "total": {
+                    "bytes": 2088960
+                },
+                "used": {
+                    "bytes": 622592,
+                    "pct": 29.80392074584961
+                }
+            },
+            "stats": {
+                "active_size": {
+                    "bytes": 135167.737856
+                },
+                "backup_time": "1900-01-01T00:00:00Z",
```

Comment:
I wonder if we could leave it empty if it has this zero-like value. If someone tries to use it in dashboards or queries they will have to clean the data, we could already do it here.

Reply:
That was the exact value returned by the database without parsing. I mean, it's the value that an experienced MSSQL user will expect to receive because it's the exact value that you get when you run the query with `sqlcmd` or with any database explorer. 

Let's try to leave our personal preferences or modifications aside because this kind of changes could confuse the users.

Please, this have been like this for days if not weeks, next time, try to make this comments earlier.